src/quantization.cpp

```diff
@@ -170,13 +308,13 @@ void quantize_int8(program& prog,
                    const std::vector<std::string>& ins_names,
                    const std::vector<std::pair<float, float>>& quant_params)
 {
-    for(size_t i = 0; i < quant_params.size(); i++)
-    {
-        auto param = quant_params.at(i);
-        std::cout << "index = " << i << ", scale = " << param.first << "\t" << param.second
-                  << std::endl;
-    }
-    std::cout << std::endl;
+    // for(size_t i = 0; i < quant_params.size(); i++)
+    // {
+    //     auto param = quant_params.at(i);
+    //     std::cout << "index = " << i << ", scale = " << param.first << "\t" << param.second
+    //               << std::endl;
+    // }
+    // std::cout << std::endl;
```

Comment:
Should this be guarded by an env variable?

Reply:
fixed